upcoming-release-notes/1930.md

```diff
@@ -0,0 +1,6 @@
+---
+category: Enhancements
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏
```suggestion
category: Bugfixes
```